app/packs/src/decidim/vocdoni/admin/steps/vote.js

```diff
@@ -1,41 +1,67 @@
-const ELECTION_VOTES_SELECTOR = ".js-votes-count";
-// Wait for 30s between stats refresh
+// Interval time in milliseconds for refreshing the statistics
 const WAIT_TIME_MS = 30000;
 
+// Fetch the votes from the API
 export const getElectionResults = async () => {
-  const vocdoniClientMetadata = document.querySelector(".js-vocdoni-client");
-  const resultsPath = vocdoniClientMetadata.dataset.resultsPath;
-  const response = await fetch(resultsPath);
-  const result = await response.json();
-  return result;
+  const vocdoniClientElement = document.querySelector(".js-vocdoni-client");
+  if (vocdoniClientElement) {
+    const resultsPath = vocdoniClientElement.dataset.resultsPath;
+    if (resultsPath) {
+      const response = await fetch(resultsPath);
+      return response.json();
+    }
+  }
+  return null;
 };
 
-// Fetch the votes from the API and show them in the UI
-const fetchTheVotesStats = async () => {
-  const electionVotesMetadataTable = document.querySelector(ELECTION_VOTES_SELECTOR);
-  if (!electionVotesMetadataTable) {
-    return;
+// Update the DOM with the answer for a specific question index
+const updateDomWithAnswer = (questionIdx, answer, value) => {
+  const dom = document.querySelector(`td[data-question-idx="${questionIdx}"][data-answer-value="${value}"]`);
+  if (dom) {
+    dom.innerHTML = answer;
   }
+};
+
+// Fetch and display vote statistics for a given question index
+const fetchAndDisplayVotes = async (questionIdx) => {
+  const { election } = await getElectionResults();
+  election[questionIdx]?.forEach(updateDomWithAnswer.bind(null, questionIdx));
+};
 
-  // Fetch the votes stats from the Vocdoni API
-  const fetchVotesStats = async () => {
-    const { election } = await getElectionResults();
-    console.log("ELECTION METADATA => ", election);
-    console.group("Partial Results");
-    election.forEach((question, idx) => {
-      question.forEach((answer, value) => {
-        const dom = document.querySelector(`td[data-question-idx="${idx}"][data-answer-value="${value}"]`);
-        console.log(`Question ${idx} - ANSWER ${value} VOTES ${answer}`, dom);
-        dom.innerHTML = answer;
-      });
-    });
-    console.groupEnd();
-  };
+// Process each accordion element to fetch and display stats if it's open
+const processAccordion = async (accordion) => {
```

Comment:
These two condictionals can be refactored to use a guard clause. I think it'd be cleaner